views/subscribers/importExport/import.html

```diff
@@ -113,7 +113,7 @@ <h1 class="title">
 'pickListsDescription': __('Pick the list that you want to import these subscribers to.'),
 'select': _x('Select', ' Verb'),
 'createANewList': __('Create a new list'),
-'updateExistingSubscribers': __("Update existing subscribers' information"),
+'updateExistingSubscribers': __("Update existing subscribers’ information, like first name, last name, etc."),
```

Comment:
```suggestion
'updateExistingSubscribers': __('Update existing subscribers’ information, like first name, last name, etc.'),
```
After Pavel's suggestion, you can replace double-quotes with the apostrophes. What do you mean?

Reply:
Done.